www/js/app.js

```diff
@@ -382,13 +383,17 @@ define(['jquery', 'zimArchiveLoader', 'uiUtil', 'settingsStore','abstractFilesys
         document.getElementById('formArticleSearch').style.display = '';
         document.getElementById('welcomeText').style.display = '';
         // Give the focus to the search field, and clean up the page contents
-        $("#prefix").val("");
-        $('#prefix').focus();
-        $("#articleList").empty();
-        $('#articleListHeaderMessage').empty();
+        document.getElementById('prefix').value = '';
+        document.getElementById('prefix').focus();
+        let articleList = document.getElementById('articleList');
+        let articleListHeaderMessage =  document.getElementById('articleListHeaderMessage');
+        while(articleList.firstChild) articleList.removeChild(articleList.firstChild);
+        while(articleListHeaderMessage.firstChild) articleListHeaderMessage.removeChild(articleListHeaderMessage.firstChild);
         document.getElementById('searchingArticles').style.display = 'none';
         document.getElementById('articleContent').style.display = 'none';
-        $("#articleContent").contents().empty();
+        let articleContent = document.getElementById('articleContent');
```

Comment:
This isn't quite right. The jQuery function `contents()` gets all the child nodes, I think, of the iframe (in this case). You'll need to iterate over them and empty them all. If you don't feel confident about this, you can revert this back to the jQuery function, and we can deal with it in a different issue/PR.

Reply:
Thinking about it, it's possible your code has the same effect as the original jQuery statement, as it would iterate over every child node anyway. It needs to be checked that we end up with a completely empty/purged iframe.

Reply:
OK, I've determined this code doesn't ever run, because articleContent is the iframe, and the iframe doesn't have any child elements. We need to select the content document. I'll make a suggestion.